### PR TITLE
Added promo card Small Asteroid (#539)

### PR DESCRIPTION
**Pull request by gomfunkel**

### CODE REVIEW

File: src/cards/promo/SmallAsteroid.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export class SmallAsteroid implements IProjectCard {
+
+    public cost: number = 10;
+    public name: CardName = CardName.SMALL_ASTEROID;
+    public tags: Array<Tags> = [Tags.SPACE];
+    public cardType: CardType = CardType.EVENT;
+
+    public play(player: Player, game: Game) {
+        game.addResourceDecreaseInterrupt(player, Resources.PLANTS, 2);
```
Comment (alrusdi) [inline src/cards/promo/SmallAsteroid.ts:17]: Can you please add test if player's own plants will not be removed in solo mode?

Comment (gomfunkel) [reply src/cards/promo/SmallAsteroid.ts:17]: Done.

**Comment by gomfunkel** [comment]
@alrusdi This one should be OK to merge now.

File: src/cards/promo/SmallAsteroid.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { IProjectCard } from '../IProjectCard';
```
[the remark this answers is not in the document]
Comment (gomfunkel) [reply src/cards/promo/SmallAsteroid.ts]: @alrusdi Fixed!

**Pull request merged**